.travis.yml

```diff
@@ -6,6 +6,11 @@ compiler:
   - clang
   - gcc
 
+addons:
+  homebrew:
+    packages:
+	- icu4c
```

Comment:
Surely 6 spaces would do?

Reply:
I'll try that again but Travis didn't seem to like it before

Reply:
for some reason when I put 6 spaces instead of a tab, Travis doesn't install ICU on Mac.